### PR TITLE
uart loop cmsis

### PR DESCRIPTION
**Pull request by romatsp**

### CODE REVIEW

File: Lesson5/Ex1/ex1_5.c [hunk 1]
```diff
@@ -1,49 +1,31 @@
...
-		         USART_CR1_TE | USART_CR1_RE | // TX ON, RX ON
-                 USART_CR1_RXNEIE;             // enable INT
-
-
-NVIC_EnableIRQ (USART2_IRQn); //UART2 INT ON
+   init_func();
+   open_func();
 
-__enable_irq ();              //Gobal INT ON
+    NVIC_EnableIRQ (USART2_IRQn); //UART2 INT ON
```
Comment (AxelP) [inline Lesson5/Ex1/ex1_5.c]: This line should go to init_func()

File: Lesson5/Ex1/ex1_5.c [hunk 2]
```diff
@@ -1,49 +1,31 @@
...
+    __enable_irq ();              //Gobal INT ON
```
Comment (AxelP) [inline Lesson5/Ex1/ex1_5.c]: This would be better to move above init_func()

File: Lesson5/Ex1/ex1_5.c (hunk 1 above)
Comment (romatsp) [reply Lesson5/Ex1/ex1_5.c]: Ok

File: Lesson5/Ex1/ex1_5.c (hunk 2 above)
Comment (romatsp) [reply Lesson5/Ex1/ex1_5.c]: Ok

**Pull request merged**